src/pages/instances/InstanceOverviewDevices.tsx

```diff
@@ -52,6 +64,53 @@ const InstanceOverviewDevices: FC<Props> = ({ instance }) => {
           role: "rowheader",
           "aria-label": "Type",
         },
+        {
+          content: (
+            <div>
+              {device.type == "disk" ? (
+                isRootDisk(device as FormDevice) ? (
+                  <>
+                    <strong>Pool: </strong>
+                    <>{device.pool}</>
+                  </>
+                ) : (
+                  <>
+                    <strong>Storage Volume:</strong>
+                    <Link
+                      to={`/ui/project/${instance.project}/storage/pool/${device.pool}/volumes/custom/${device.source}`}
+                    >
+                      {` ${device.source}`}
+                    </Link>
+                  </>
+                )
+              ) : (
+                ""
+              )}
+              {device.type === "gpu" && (
+                <>
+                  <strong>Pci: </strong>
+                  <>{device.pci}</>
+                </>
+              )}
+              {device.type === "proxy" &&
+                deviceDetailRows([
+                  `Listen: ${device.listen}`,
+                  `Connect: ${device.connect}`,
+                ])}
+              {device.type === "usb" ? `uid: ${device.uid}` : ""}
+              {device.type === "infiniband"
+                ? `Nic type: ${device.nictype}`
+                : ""}
+              {device.type === "pci" ? `Address: ${device.address}` : ""}
+              {device.type === "tpm" ? `Path: ${device.path}` : ""}
+              {device.type === "unix-block" ? `uid: ${device.uid}` : ""}
+              {device.type === "unix-char" ? `uid: ${device.uid}` : ""}
+              {device.type === "unix-hotplug" ? `uid: ${device.uid}` : ""}
+            </div>
+          ),
+          role: "rowheader",
+          "aria-label": "Details",
```

Comment:
we should handle the case if a value is too long for the table cell. You should be able to add `className` here with `u-truncate`. Then you would also need to include a `title` attribute to display device value on hover of the cell.